spec/restaurant_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'restaurant'
+require 'order'
```

Comment:
Bit of overlap in unit testing here, but I can see a lot of functionality from Order class is needed to make sure that the Restaurant class is executing it properly. Perhaps that's a problem? However, I think your classes are generally set up very well